SparkConnector/setup.py

```diff
@@ -71,7 +71,6 @@
     cmdclass= cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab>=4.0.0,<5",
```

Comment:
Well, it is a runtime dependency because the extension runs inside jupyterlab, right? I would say as commit message:
```
Even though SparkConnector is a lab extension, we can assume lab will be installed
when using the extension. Removing the explicit dependency helps us not pull
lab and all its dependencies when constructing a custom environment in SWAN.
```